extension/event-page.js

```diff
@@ -93,32 +97,55 @@ function updateBadge(tabId, response) {
 }
 
 function filterResponse(response) {
-
   if (!response) return response;
 
-  let ret = response.filter(entity => /^https?:\/\/schema.org\/(Product|Event)$/.test(entity['@type']));
+  let ret = response.filter(entity => {
+    if (/^https?:\/\/schema.org\/(Product|Event)$/.test(entity['@type'])) return true;
+    if (entity['@type'].endsWith('CustomEvent')) return true;
+    if (entity['@type'] == manifestType) return true;
+    return false;
+  });
   return ret;
 }
 
-function updateArc(tabId, response) {
+function loadManifest(url) {
+  return loader.loadResource(url).then(contents => {
```

Comment:
What happens if there's an error loading the url?

Reply:
A fair question, but I deleted the code.